src/site/content/en/blog/lighthouse-whats-new-6.0/index.md

```diff
@@ -0,0 +1,463 @@
+not be a sufficient replacement for some of our users. These are the key differences:
+
+-  Pagespeed Insights is unable to audit non-public websites, since it is run via a remote
+   server and not your local Chrome instance. 
+   -  If you need to audit a non-public website, use the DevTools Lighthouse panel, or the
+      Node CLI.
+
+-  Pagespeed Insights is not guaranteed to use the latest Lighthouse version.
+   -  If you want to use the latest Lighthouse version, use the Node CLI. The browser
+      extension will get the update ~1-2 weeks after release.
+
+-  Pagespeed Insights is a Google API, using it constitutes accepting the Google API Terms of
+   Service.
+   -  If you do not wish to/cannot accept the terms of service, use the DevTools Lighthouse
+      panel, or the Node CLI.
+
+The good news is that simplifying the product story allowed us to focus on other engineering
+problems. As a result, we released the [Lighthouse Firefox
+extension](https://addons.mozilla.org/en-US/firefox/addon/google-lighthouse/)!
+
+## Budgets {: #budgets }
+
+Lighthouse 5.0 introduced [performance budgets](https://web.dev/performance-budgets-101/) which
+supported adding thresholds for
+[how much of each resource type](https://github.com/GoogleChrome/lighthouse/blob/master/docs/performance-budgets.md#resource-budgets)
+(such as scripts, images, or css) a page can serve.
+
+Lighthouse 6.0 adds
+[support for budgeting metrics](https://github.com/GoogleChrome/lighthouse/blob/master/docs/performance-budgets.md#timing-budgets),
+so now you can set thresholds for specific metrics such as FCP. For now, budgets are only available
+to the Node CLI and Lighthouse CI.
+
+## Source location links {: #source-location }
+
+Some of the issues that Lighthouse finds about a page can be traced back to a specific line of
+source code and the report will state the exact file and line that's relevant. To make this easy to
+explore in DevTools, clicking on the locations specified in the report will open the relevant files
+in the **Sources** panel.
+
+<figure>
+  <video autoplay loop muted playsinline>
+    <source src="./lighthouse-source-location.webm" type="video/webm">
+    <source src="./lighthouse-source-location.mp4" type="video/mp4">
+  </video>
+  <figcaption>
+    DevTools reveals the exact line of code that causes the issue.
+  </figcaption>
+</figure>
+
+## On the horizon {: #horizon }
+
+Lighthouse has begun experimenting with collecting source maps to power new features, such as:
+
+-  Detecting duplicate modules in JavaScript bundles.
+-  Detecting excessive polyfills or transforms in code sent to modern browsers.
+-  Augmenting the Unused JavaScript audit to provide module-level granularity.
+-  Treemap visualizations highlighting the modules that require action.
+-  Displaying the original source code for report items with a "source location".
+
+![image](lighthouse-sourcemaps-unusedjs.png)
```

Comment:
because the matte is white i think this needs a border.

also... maybe just delete a few rows in here with devtools so it can be shorter. feels a bit too tall.

Reply:
may also need alt text for the image